api/v1alpha1/nifiparametercontext_types.go

```diff
@@ -17,6 +17,8 @@ type NifiParameterContextSpec struct {
 	ClusterRef ClusterReference `json:"clusterRef,omitempty"`
 	// a list of secret containing sensitive parameters (the key will name of the parameter).
 	SecretRefs []SecretReference `json:"secretRefs,omitempty"`
+	// a list of references to NiFiParameterContext
+	ParameterContextRefs []ParameterContextReference `json:"parameterContextRefs,omitempty"`
```

Comment:
This is really minor, but do you think these should be called `inheritedParameterContextRefs` to match the NiFi API? 

I suppose at least the comments & docs should clarify these are inherited parameter context refs.